mlir/lib/Dialect/GPU/Utils/DistributionUtils.cpp

```diff
@@ -83,7 +83,7 @@ WarpDistributionPattern::moveRegionToNewWarpOpAndAppendReturns(
 
 OpOperand *WarpDistributionPattern::getWarpResult(
     WarpExecuteOnLane0Op warpOp,
-    const std::function<bool(Operation *)> &fn) const {
+    const llvm::function_ref<bool(Operation *)> fn) const {
```

Comment:
```suggestion
    llvm::function_ref<bool(Operation *)> fn) const {
```

Reply:
this looks like some funny github bug, the code was different at the time you left this comment :)